docs/upgrade.html

```diff
@@ -47,14 +47,77 @@
 
 <p><b>Note:</b> Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
-<ol>
+<h5><a id="upgrade_1020_streams" href="#upgrade_1020_streams">Updating a Kafka Streams Application</a></h5>
+<ul>
     <li>Upgrading a Kafka Streams Applications:
         <ul>
-            <li>You need to recompile your code. Just swapping the jar file will not work and will break your appliation.</li>
-            <li>If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed.</li>
+            <li> You need to recompile your code. Just swapping the jar file will not work and will break your application. </li>
+            <li> <code>KStreamBuilder#table()</code> always requires to specify a store name. </li>
+            <li> <code>KTable#through()</code> always requires to specify a store name. </li>
+            <li> If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed. </li>
+            <li> If you register custom metrics, you will need to update this code, because the <code>StreamsMetric</code> interface was changed. </li>
+            <li> Upgrading your Streams application from 0.10.1 to 0.10.2 does not require to upgrade your brokers.
+                 A Kafka Streams 0.10.2 application can connect to 0.10.2 and 0.10.1 brokers (it is not possible to connect to 0.10.0 brokers though). </li>
         </ul>
     </li>
-</ol>
+
+    <li>Streams API changes in 0.10.2.0:
+         <li> New methods in <code>KafkaStreams</code>:
```

Comment:
This looks a bit out of place here. The upgrade doc tends to focus on things that have a compatibility impact. Maybe API improvements should be listed somewhere else?